14_team6_agentdesign/index.tex

```diff
@@ -1 +1,422 @@
+\begin{algorithm}[!h]
+\SetAlgoLined
+    \uIf{Personality == Selfish}{
+        request gifts scaled up by friendship coefficients\;
+    }
+    \uElseIf{Personality == Normal}{
+        request gifts without scaling\;
+    }
+    \uElseIf{Personality == Generous}{
+        request gifts scaled down by friendship coefficients\;
+    }
+    \caption{Gifts Requests}
+\end{algorithm}
+
+\begin{algorithm}[!h]
+\SetAlgoLined
+    \uIf{Personality == Selfish}{
+        request gifts scaled down by friendship coefficients\;
+    }
+    \uElseIf{Personality == Normal}{
+        request gifts without scaling\;
+    }
+    \uElseIf{Personality == Generous}{
+        request gifts scaled up by friendship coefficients\;
+    }
+    \caption{Gifts Offers}
+\end{algorithm}
+
+
+The algorithm above concludes how our agent makes use of Friendship and Personality, and behaves differently during the gift exchanging section. Basically, the amount of any gifts can be linearly scaled based on how much friendship we have with the target island whom we are requesting or offering gifts to. For example, if our island is \emph{selfish}, the amount of the gift we offer will be scaled down by a value depending on the current friendship. A relative coefficient will be calculated based on that. If the island is a good friend with us, the coefficient will be relatively lower when it comes to the gift offers, so that the amount by which it is scaled down will be less. Otherwise, the coefficient will be higher and a higher penalty will be applied so that we offer our bad friend very few gifts.
+%\subsubsection{Opinions}
+
+\subsection{IIFO} \label{subsec:Team6_IIFO}
+\subsubsection{Long-Term Collective Risk Dilemma} \label{subsubsec:Team6_IIFO:ltCRD}
+% Individual utilities vs CPR
+Our agent faces a cost of living deducted at every turn, and an expenditure when a disaster hits. To mitigate the effect of disaster, islands make contributions to the common-pool. The long-term collective risk dilemma (ltCRD) defines that how much resources are needed to be raised and before when. We parameterise such problem with a threshold $T$ and a deadline day $D$.
+
+
+\subsubsection{Strategies} \label{subsubsec:Team6_IIFO:Strategies}
+According to Chapter 4 Environment, there are non-stochastic and stochastic disasters. For the non-stochastic disaster, $T$ is known and $D$ is unknown. For the stochastic disaster, both $T$ and $D$ are unknown. To deal with ltCRD, the islands make forecast about $T$ and $D$, whereas each island has its own knowledge to predict the disaster. On the one hand, when an individual island forms an opinion regarding the desired contribution to the common-pool resources, it is necessary to convince others that the decision is rational. On the other hand, we take the aggregate knowledge into consideration, taking advantages of distributed computation of disaster prediction. Such prediction information comprises:
```

Comment:
you cannot do like this.. please refer to it appropriately.

Reply:
refer to the chapter by `\ref{}` thingy set in the guidelines

Reply:
actually didn't find their label for it - will replace it with using a label ref

Reply:
you can do multiple PRs :)
- PR about the label
- then update this PR of yours

Reply:
will do it 7 hrs later